rtloader/common/stringutils.c

```diff
@@ -8,30 +8,36 @@
 #include "rtloader_types.h"
 #include "stringutils.h"
 
-
-PyObject * yload = NULL;
-PyObject * ydump = NULL;
-PyObject * loader = NULL;
-PyObject * dumper = NULL;
+PyObject *yload = NULL;
+PyObject *ydump = NULL;
+PyObject *loader = NULL;
+PyObject *dumper = NULL;
```

Comment:
Please don't reformat the code you did not change.

Reply:
I didn't do it actually, not directly or willingly at least . Kind of. My changes to the file failed CLANG formatting tests in CI and I was forced to run CLANG formatting on the file. I have just tested by putting original formatting in place and re-running CLANG template and it did change original code to the current formatting. I am not sure why this was not done earlier or perhaps some CLANG formatting change and I theoretically can put original code back but it may suffer the same fate next time.